src/meep.hpp

```diff
@@ -714,6 +714,7 @@ class structure {
   grid_volume *effort_volumes;
   double *effort;
   int num_effort_volumes;
+  double t_setepsilon, t_setconductivity, t_chunkdivision, t_addsusceptibility;
```

Comment:
This should be done by adding new `time_sink` types rather than explicit fields.